languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -34110,6 +34110,76 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TYPOGRAPHIE" name="Typographie" type="typographical">
+      <rulegroup id="PAS_DE_VIRGULE" name="pas de virgule" default="temp_off">
+          <rule>
+              <pattern>
+                  <marker>
+                      <token>,</token>
+                      <token regexp="yes">et|ou</token>
+                  </marker>
+              </pattern>
+              <message>La virgule doit être omise dans ce contexte.</message>
+              <suggestion>\2</suggestion>
+              <example correction="et">Elle a acheté des radis<marker>, et</marker> des choux.</example>
+          </rule>
+          <rule>
+              <pattern>
+                  <token>c'</token>
+                  <token skip="-1">est</token>
+                  <marker>
+                      <token>,</token>
+                      <token regexp="yes">qui|que</token>
+                  </marker>
+              </pattern>
+              <message>La virgule doit être omise dans ce contexte.</message>
+              <suggestion>\4</suggestion>
+              <example correction="qui">C'est lui<marker>, qui</marker> part demain.</example>
+              <example correction="que">C’est l’eau laissée dans la piscine<marker>, que</marker> j'ai vidée.</example>
+          </rule>
+          <rule>
+              <antipattern>
+                  <token postag="SENT_START"/>
+                  <token regexp="yes">lui|elle|eux|nous|vous|elles</token>
+                  <token regexp="yes">seul?|-même</token>
+                  <token>,</token>
+                  <token postag="V.*" postag_regexp="yes"/>
+                  <token>-t-il</token>
+              </antipattern>
+              <pattern>
+                  <token postag="SENT_START"/>
+                  <token regexp="yes">lui|elle|eux|nous|vous|elles</token>
+                  <marker>
+                      <token regexp="yes">seul?|-même</token>
```

Comment:
Why `seul?` ? This will match `seul` and `seu`. Did you mean `seule?s?`?

Problem is, "moi-même", "elle-même", "elles-mêmes" etc. are all just one token. So you have to put that into the token before via regex (`lui(-même)?` etc.)